src/comments.ts

```diff
@@ -120,6 +121,20 @@ export const WaitUntilMergeIsOK = (user: string, abbrOid: string, uri: string, m
         |Thanks, and happy typing!`
 });
 
+export const RemindPeopleTheyCanUnblockPR = (user: string, approvalUsers: string[], abbrOid: string) => ({
+    // at most one reminder per update
+    tag: `wait-for-merge-offer-${abbrOid}`,
+    status: txt`
+        |:hourglass_flowing_sand: Hi @${user},
+        |
+        |It's been a few days since this PR was approved by ${approvalUsers.join(", ")}
+         and we're waiting on a DT maintainer to give a review.
+        |
+        |If you would like to short-circuit this wait, you can edit some of the [test files](${urls.testsTs})
+         in the package which verify how the \`.d.ts\` files work. This would allow the PR to be
+         merged by you or the DT module owners.`
```

Comment:
(I added a url constant and made the source more readable.)

Reply:
Thanks - yeah, I'll add "after a re-review" and I think this is good then

Reply:
Yeah, that's a good addition